### PR TITLE
Update bug-report.md to mention .NET5 SDK as the primary source for analyzers

### PR DESCRIPTION
**Pull request by mavasani**
Also moved the `Analyzer` section near the top

### CODE REVIEW

**Comment by mavasani** [comment]
Also tagging @jeffhandley @terrajobst - please let me know if you'd like the template to be structured differently or request more information (like say their project TFM, supported platforms, etc.)

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -7,6 +7,22 @@ assignees: ''
...
+
+### Analyzer source
+
+**SDK**: [Built-in CA analyzers in .NET5 SDK or later](https://docs.microsoft.com/dotnet/fundamentals/productivity/code-analysis)
+
+**Version**: [.NET5 Preview8](https://dotnet.microsoft.com/download/dotnet/5.0)
+
+_OR_
+
+**NuGet Package**: [Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)
```
Comment (mavasani) [inline .github/ISSUE_TEMPLATE/bug-report.md:22]: I will change this to `Microsoft.CodeAnalysis.NetAnalyzers` once that package is released on nuget. @jmarolf I think we should release 5.0.0-rc1 version of the package once .NET5 RC1 goes out

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -7,6 +7,22 @@ assignees: ''
...
 
+### Analyzer
+
+**Diagnostic ID**: [CA2013](https://docs.microsoft.com/visualstudio/code-quality/ca2013): `Do not use ReferenceEquals with value types`
+
+### Analyzer source
+
+**SDK**: [Built-in CA analyzers in .NET5 SDK or later](https://docs.microsoft.com/dotnet/fundamentals/productivity/code-analysis)
+
+**Version**: [.NET5 Preview8](https://dotnet.microsoft.com/download/dotnet/5.0)
```
Comment (jeffhandley) [inline .github/ISSUE_TEMPLATE/bug-report.md:18]:
```suggestion
**SDK**: [Built-in CA analyzers in .NET 5 SDK or later](https://docs.microsoft.com/dotnet/fundamentals/productivity/code-analysis)

**Version**: [SDK 5.0.100-preview.8](https://dotnet.microsoft.com/download/dotnet/5.0)
```

**Comment by mavasani** [comment]
Thanks!

**Pull request merged**